app/views/bookings/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for [@dog, @booking] do |f| %>
```

Comment:
i think you can just do a render form here to use the _form you created in the partial file (right now it's replicated)